RELEASING.md

```diff
@@ -26,3 +26,54 @@
         helm repo update elastic
         helm search repo -l elastic/apm-attacher
         ```
+
+## Testing procedure (for Linux, Windows works too but you need to adjust how files are created or create them manually)
+
+1. Clone (or update) the repo locally
+    - `git clone https://github.com/elastic/apm-k8s-attacher.git`
+2. `cd apm-k8s-attacher`
+3. Create the custom values ymal file - replacing the secret token and url with valid values for a server is better, buy even with these dummy values testing still works, just the agent won't connect to a server
+```
+cat > custom.yaml <<EOF
+apm:
+  secret_token: SuP3RT0K3N 
+  namespaces: 
+    - default
+webhookConfig:
+  agents:
+    java: 
+      environment:
+        ELASTIC_APM_SERVER_URL: "https://apm-example.com:8200" 
+        ELASTIC_APM_ENVIRONMENT: "prod"
+        ELASTIC_APM_LOG_LEVEL: "info"
+EOF
+```
+4. Install the attacher - note the namespace can be any namespace but can't be the default nor a namespace where pods will be tested
+    - `helm install test-main charts/apm-attacher --values custom.yaml --namespace=elastic-apm --create-namespace` 
```

Comment:
Should this add the build step before? `make .webhook` I think it was.

Reply:
The current testing steps are mixed: the *released* `docker.elastic.co/observability/apm-attacher` image, but the using the *local* "./charts/..." directory for the chart (rather than using the published chart name `elastic/apm-attacher`).

Reply:
Yes, you are entirely right. I'm going to release with this but we should fix that testing doc before a future release if any